File: Data/Models/Carer.cs
namespace Blackwood.Access.Models {
	using System;
	using System.ComponentModel.DataAnnotations;

	public class Carer {
		public string PersonnelNumber { get; set; }
		[Key]
		public int CarerCode { get;set; }
		public string Forename { get; set; }
		public string Surname { get; set; }
		public string Email { get; set; }
		public Guid? CareSysGuid { get; set;}
		public Int16? DefaultTeamCode { get; set; }
	}
}

File: ClientApp/app/models/payroll/Carer.ts
import { Guid } from "../Utilities";
import { Team } from "./Team";
import { CarerContract } from "./Contract";

export class Carer {
	public personnelNumber: string;
	public carerCode: number;
	public forename: string;
	public surname: string;
	public email: string;
	public careSysGuid: Guid;
	public defaultTeamCode: number;

	public defaultTeam: Team;

	public contracts: CarerContract[];
}